frontend/src/components/Tree/Tree.jsx

```diff
@@ -92,7 +102,22 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick) {
     .attr("text-anchor", "middle")
     .attr("font-size", 18)
     .attr("y", -15)
-    .text((node) => node.data.name);
+    .attr(
+      "transform",
+      (d) => `
+        rotate(${(d.x * 180) / Math.PI - 90}) 
+        translate(${d.y},0) 
+        rotate(${d.x >= Math.PI ? 180 : 0})
+      `
+    )
+    .attr("dy", "0.90em")
+    .attr("dx", "0.0em")
+    // eslint-disable-next-line
+    .attr("x", d => d.x < Math.PI === !d.children ? 6 : -6)
```

Comment:
Please write a comment explaining this line

Reply:
Ah, this line makes such a small adjustment that it is not worth keeping around; removed, thanks!

Reply:
Added comment